bettercli/cl/date.py

```
@@ -0,0 +1,316 @@
+
+    def __getitem__(self, key: t.Any) -> t.Any:
+        if key not in self or isinstance(key, (slice, int)):
+            return self.list[key]
+        return super().__getitem__(key)
+    
+
+
+class Date:
+    cache = Cache()
+    months = Dict({'January':31, 'February':28, 'March':31, 'April':30, 'May':31, 'June':30, 'July':31, 'August':31, 'September':30, 'October':31, 'November':30, 'December':31})
+    def __init__(self, style:'STYLE', min:'t.Union[d, None]'=None, max:'t.Union[d, None]'=None, default:'d'=None) -> None:
```

Ruff (0.8.2)

Undefined name `STYLE`

(F821)